tools/archiver.py

```diff
@@ -920,6 +935,11 @@ def main():
         help="If pushing to ElasticSearch fails, dump documents in JSON format to this directory and "
         "fail silently.",
     )
+    parser.add_argument(
+        "--defaultdate",
```

Comment:
Given that the value is treated as an epoch, why no call it defaultepoch?
There might be a use-case for the defaulting to the current time: that is no longer possible.

Given that epoch cannot be negative, maybe use -1 for skip and -2 for current time?

Reply:
it's currently 'skip' to skip, and leave out the arg to use default behavior which is current date. I think this is reasonable.